src/json_value_module.F90

```diff
@@ -2579,6 +2729,59 @@ subroutine wrap_json_value_add_double_vec(json, p, name, val)
     call json%add(p, to_unicode(name), val)
 
     end subroutine wrap_json_value_add_double_vec
+
+!*****************************************************************************************
+!> author: Jacob Williams
```

Comment:
I really just did copy/paste and changed a few kinds, so no need to add my name, unless it helps to better track the work